### DISCUSSION TITLE
SI4463 availability

### DISCUSSION THREAD

**Discussion by jrcardwell** [peterantypas/maiana#24] [Ideas]
Since the SI 4463 won’t be in stock until Aug or Sep 2021, what about using the 4460 which does seem to be available.  Would it have reduced functionality? Great project by the way -  I’d definitely be interested in the kit!

**Comment by peterantypas** [comment]
The 4463 in my design configuration outputs +16dBm and the PA adds 17dB of gain. I suspect that we'll end up losing at least 3dB (maybe more) across the board, and that's the best case scenario. I may have to switch to the AFT05MS004NT1 for the PA, which could give us 20dB of gain according to the data sheet. So basically, to achieve the same transmit power, I will need to do a complete redesign of the TX section.

I have a sizeable order for the 4463 placed, so I do expect some quantity in Q1 of 2021. Also, I just tested the Si4362 into the second RF IC position (that chip is always a receiver) and it's a drop-in replacement with no firmware changes. So now each board needs one 4463 instead of 2. 

I'll order some 4460s and play with them to see what happens ...

**Comment by astuder** [comment]
FWIW, the Si4467 is back in stock at Mouser and Digi-Key

**Reply by peterantypas** [reply]
It looks like these two are basically equivalent for this application. They'll give us +10dBm in Class-E matching. So total output power as it stands now will end up at +27dBm (1/2 Watt). Even with the higher gain AFT05MS004NT1, we won't go past 1 Watt. We'll need an intermediate gain stage with about 6dB of gain. A single BJT in class A will probably do it ... I'll think about it.

**Reply by astuder** [reply]
Si4468 is the more powerful variant, but that one's in short supply too.

**Comment by peterantypas** [comment]
Looking at the AFT05MS003NT1 data sheet again, it might be possible to get close to +33dBm if we push the bias voltage higher. Worth a try.

![Screenshot from 2020-12-14 06-32-48](https://user-images.githubusercontent.com/1565933/102093775-6616d480-3dd6-11eb-9f0f-ec29b8379436.png)

**Comment by peterantypas** [comment]
Another option is to drive down the Si44xx and inject a gain block like this:

https://www.ti.com/lit/ds/symlink/trf37a73.pdf?HQS=TI-null-null-digikeymode-df-pf-null-wwe&ts=1607957225975

This would allow us to use any Silabs IC in this series with an output no higher than +6dBm. It takes 3.3V, so an SPDT load switch could alternate between powering the RX LNA vs this TX gain block. I'm going to look into this to see if I can lay it out without disturbing the test points, as the jig is already built and working:


![MAIANA-Jig](https://user-images.githubusercontent.com/1565933/102097256-85affc00-3dda-11eb-9f09-b7ea70cb0fbc.jpg)

**Reply by peterantypas** [reply]
I'm making a breakout to try out this TI part with the rest of the backend. Hopefully it solves the problem.

**Reply by peterantypas** [reply]
I've had this breakout for the Si4463 back end for a while now, so I finally built it last night. It allows me to transmit long (randomly) modulated packets for power measurement and spectrum analysis on demand (via a CLI). I'm glad I finally did this because it turned out that the output of the existing design and firmware is only +12.5dBm in class E, which means the current PA adds more than 20dB of gain.

I also have a separate breakout for the PA with independent drain and bias voltages, so this will allow me to test both power MOSFETs and see what can be done with 9-10dBm from the back end. Maybe we don't need the TI part.

**Comment by peterantypas** [comment]
We're in luck. With the back end configured at exactly 10dBm output, the PA gate bias at 3.3V and the drain voltage at 9V, I was able to get exactly +33dBm output (there is a 10dB attenuator at the SA's input):

![10dBmDrive](https://user-images.githubusercontent.com/1565933/102301099-0837d900-3f0b-11eb-8be1-f97944ae355b.jpg)

**Comment by peterantypas** [comment]
A further refinement in the PA (increasing the drain choke inductance) delivered +34dBm, so now the drain voltage is back at 8V and the gate bias at 3V. Drive level is still +10dBm.

If the 4460/4467 can actually output more than 9dBm, we should be good. I'm ordering both to try.

**Comment by peterantypas** [comment]
This is nominally impossible according to the data sheet, but I'll take it !!!
![Si4460](https://user-images.githubusercontent.com/1565933/102570729-d0609b00-409c-11eb-9880-628454f2561e.jpg)

**Reply by peterantypas** [reply]
Alright, done! More than +35dBm output with 8V on the drain and 3V bias on the gate. Had to tone the power level down in firmware.

So this is the solution:

- I will characterize the 4463, 4467 and 4460
- Firmware will send the PART_INFO command to retrieve the exact model of the transceiver IC during initialization
- Based on the model, a lookup table will determine the exact PA settings for a total conducted output of +33dBm
- The circuit will change slightly to allow a higher bias voltage on the MOSFET gate (different voltage divider). This is just a BOM change, board layout stays the same

**Reply by peterantypas** [reply]
Si4467 threw a bit of a wrench, as it needs a very different configuration sequence from the others. So now I am going to identify the chip first, then initialize it with a chip-specific command array from Wireless Studio, then configure the power level accordingly.

Once I generated new code for the 4467, it achieved exactly the same output as the 4460 with the same PA settings.

**Reply by astuder** [reply]
I do the same in the dAISy receivers. Though IIRC the Si4467 also worked when throwing the configuration generated for an Si4362 at it.

**Reply by peterantypas** [reply]
Alright, 4467 it is. Just tested with a 4362 receiver in the second slot and RX performance is nominal. TX power took only a firmware adjustment to reach +33.2dBm, so no other BOM change is needed. The good news is that Mouser has a pretty good schedule for deliveries of this, so I don't need to buy all of its stock tonight ;)

**Reply by jrcardwell** [reply]
Great news Peter!

**Comment by peterantypas** [comment]
I don’t think that this design will ever achieve full certification because the regulatory requirements on AIS devices are pretty steep and I intentionally ignored them (no DSC support for instance).

The radio is perfectly within specs and could pass radiated testing, but that’s about the extent of it.

For this reason, I will ship the kit configured as a receiver. Soldering one inductor can turn it back into a transmitter and I will leave that choice to each individual, where it belongs.

**Comment by Rudder2** [comment]
The government confuses me... Why does an AIS transponder need DSC? I would of ignored that also...

 If one wants such a thing get a VHF with it in it so that the coast guard can yell at you when your kid presses the button... That was a fun day on the radio...

I'm willing to take the risk and activate the transmit feature. Would there be an easy way to turn off the transmit like if in pirate waters? 

Really looking forward to your project coming out.

> I don’t think that this design will ever achieve full certification because the regulatory requirements on AIS devices are pretty steep and I intentionally ignored them (no DSC support for instance).
> 
> The radio is perfectly within specs and could pass radiated testing, but that’s about the extent of it.
> 
> For this reason, I will ship the kit configured as a receiver. Soldering one inductor can turn it back into a transmitter and I will leave that choice to each individual, where it belongs.
> 
> Sent from my iPhone
>

**Comment by peterantypas** [comment]
Yes, the breakout will have a "silent mode" switch.

**Reply by bergaliv** [reply]
When will the kits be avaliable to buy? And at what cost?

**Reply by peterantypas** [reply]
I am not working on MAIANA at the moment. I had to stop to rescue the project that pays my salary. It's literally a bloodbath out there.

**Comment by peterantypas** [comment]
This just came. Took about 20 weeks:

![IMG_1085](https://user-images.githubusercontent.com/1565933/118722399-fa032700-b7e0-11eb-89b9-47e128d64740.JPG)
